static/webpack-config.js

```diff
@@ -79,53 +79,37 @@ module.exports = function () {
     {
       mode,
       devtool: 'source-map',
-      target: ['web', 'es5'],
-      entry: {
-        'locator-bundle': `./${jamboConfig.dirs.output}/static/js/locator-bundle.js`
-      },
-      resolve: {
-        alias: {
-          static: path.resolve(__dirname, jamboConfig.dirs.output, 'static'),
-        }
-      },
-      output: {
-        filename: '[name].js',
-        path: path.resolve(__dirname, jamboConfig.dirs.output),
-        library: {
-          name: 'VerticalFullPageMap',
-          type: 'window'
-        },
-        publicPath: ''
-      },
-      module: {
-        rules: [javascriptModuleRule],
```

Comment:
Does this mean that now the locator-bundle uses all of the same module rules as the other bundle?

Reply:
the locator-bundle only imports .js files, so it always did!